torch/testing/_internal/common_fsdp.py

```diff
@@ -44,7 +44,7 @@ def get_full_params(model, recurse=True):
     else:
         torch.cuda.synchronize()
         model._rebuild_full_params()
-        if model.module.flat_param is not None:
+        if hasattr(model.module, "flat_param"):
```

Comment:
Does this PR remove the functionality of allowing user to access the flat param with `self.flat_param`? Do we want to support such an accessor or is that not as useful?